galaxy_ng/app/access_control/fields.py

```diff
@@ -2,51 +2,48 @@
 from django.db.models import Q
 from django.utils.translation import gettext_lazy as _
 
-from guardian.shortcuts import get_perms_for_model
 
 from rest_framework import serializers
 from rest_framework.exceptions import ValidationError
 
+from pulpcore.app.models.role import Role
+
+from pulpcore.app.role_util import get_perms_for_model
```

Comment:
@bmbouter do you agree, we should have these in the plugin api?

Reply:
I think so; if a plugin needs to use it and it's the right tool for the job then yes.